lib/ansible/plugins/lookup/password.py

```diff
@@ -276,7 +277,7 @@ def _get_lock(b_path):
             os.close(fd)
             first_process = True
         except OSError as e:
-            if e.strerror != 'File exists':
+            if e.errno != errno.EEXIST:
                 raise
```

Comment:
If we're doing this already, there's a specific exception for `EEXIST` in Python 3:
```suggestion
        except FileExistsError:
            ...
```
*(https://docs.python.org/3/library/exceptions.html#FileExistsError)*